app/controllers/student_flashcards_controller.rb

```diff
@@ -2,9 +2,13 @@ class StudentFlashcardsController < ApplicationController
   before_action :find_student_flashcard, only: [:show, :edit, :update, :destroy]
 
   def index
-    @student_flashcard_set = StudentFlashcardSet.find(params[:student_flashcard_set_id])
-    authorize @student_flashcard_set
-    @student_flashcards = policy_scope(StudentFlashcard.where(student_flashcard_set: @student_flashcard_set))
+    @student_flashcards = policy_scope(StudentFlashcard)
+    authorize @student_flashcards
+    # @student_flashcard_set = StudentFlashcardSet.find(params[:student_flashcard_set_id])
+    @flashcard_set = FlashcardSet.find(params[:flashcard_set_id])
```

Comment:
the student flashcard is not nested in flashcard set, how do you take the params from the id? and why do you need them?

Reply:
same for the flashcard template